src/components/Sidebar/Sidebar.tsx

```diff
@@ -1,29 +1,120 @@
 import React from 'react';
+import { useState } from 'react'
 import './Sidebar.scss';
 import { GlobalProps } from '../app/App';
+import logo from '../../images/logo.svg'
+import logotext from '../../images/logo-text.svg'
+import feed_normal from '../../images/feed_normal.svg'
+import feed_selected from '../../images/feed_selected.svg'
+import explore_normal from '../../images/search_normal.svg'
+import explore_selected from '../../images/search_selected.svg'
+import notif_normal from '../../images/notification_normal.svg'
+import notif_selected from '../../images/notification_selected.svg'
+import direct_normal from '../../images/direct_normal.svg'
+import direct_selected from '../../images/direct_selected.svg'
+import tv_normal from '../../images/tv_normal.svg'
+import tv_selected from '../../images/tv_selected.svg'
+import stats_normal from '../../images/stats_normal.svg'
+import stats_selected from '../../images/stats_selected.svg'
+import settings_normal from '../../images/settings_normal.svg'
+import settings_selected from '../../images/settings_selected.svg'
 import ProfileInfo from "../ProfileInfo/ProfileInfo";
+import NavItem from "../NavItem/NavItem";
 
-
+export enum Navigation {
```

Comment:
Maybe rename to NavigationType ?
We'll normall yuse pure enity name for objects, like
Feed {}, or User {},

Reply:
I see your argument, I will change it, your suggestion seems better.

Just a thought, don't you think **NavigationType** seems more like an entity, as **Navigation** is a verb?